packages/concerto-core/lib/decoratormanager.js

```diff
@@ -813,3 +813,4 @@ class DecoratorManager {
 }
 
 module.exports = DecoratorManager;
```

Comment:
```suggestion
module.exports = { DCS_MODEL, DecoratorManager };
```

Reply:
Can you please export this using the [index.js](https://github.com/accordproject/concerto/blob/8997f3ce9f240b3d71471b5c78166e2e6a7e9c96/packages/concerto-core/index.js#L94)?

Reply:
But doing so will give an error as DecoratorManager is supposed to be exported default in tests that are written

Reply:
Hmmm, I'm wondering we could just fix the tests with imports, but could this could break for someone whose not importing through `index.js`. Let me discuss this during today's WG call.

Reply:
I just discussed, we decided to move `DCS_MODEL` to `concerto-metamodel` package. I'll raise a PR and cut a release and update it here to be used.

Can you please wait until I move things there?

Reply:
@sanketshevkar yeah sure , no problem . I would like to contribute , could you direct me to something where i could ?

Reply:
Thanks @Ayush1404 , you'll have to move the DCS model in this [repo](https://github.com/sanketshevkar/concerto-metamodel) and import it in core and types package in concerto.

Reply:
@sanketshevkar , okay do i do it here in this PR or open a separate issue for it ? Also i meant to ask where else could i contribute to concerto or accord project in general .